docs/resourcebookings/resourcebookings.rst

```diff
@@ -0,0 +1,143 @@
+﻿Reserveringen van Middelen
+=============================
+
+Dit reserveringssysteem is ontworpen om het beheer en de planning van middelen te vereenvoudigen door een flexibele,
+geautomatiseerde en gebruiksvriendelijke ervaring te bieden. Of het nu via de backend of een partnerportaal gebeurt,
+gebruikers kunnen eenvoudig de beschikbaarheid bekijken, reserveringen maken en planningen beheren met minimale inspanning.
+
+**Belangrijkste functionaliteiten:**
+
+• Definieer het type reservering met een kalender die beschikbaarheid toont
+• Reserveer meerdere middelen tegelijk (alleen als ze allemaal beschikbaar zijn)
```

Comment:
Please_ use "-" rst markup, or * whatever you prefer.
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html